### PR TITLE
Insertion of new username 

### LINKED ISSUE #49: Inserting wrong ID
**Describe the bug**
When we insert values once again instead of coming back to the main activity then the `username` is not inserted in the correct way.

##### ex:
First time: `hello`
Second time: `hellohello`
Third time: `hellohellohello`

**To Reproduce**
1. Go to `insert_activity`
2. Insert any values 
3. Instead of coming back to main activity insert once again
4. See the database
5. Observe the error

**Expected behavior**
Inserting new values should happen properly.

**Smartphone (please complete the following information):**
 - Device: Samsung J8
 - OS:  Android 9 Pie

**Would you like to work on the issue?** (Yes/No)
No

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
Fixed #49 
The changes made, allow the correct insertion of the username in the database.

### CODE REVIEW

**Comment by yashk2000** [comment]
@ShivangiSingh17 please follow semantic pr guidelines.

File: app/src/main/java/org/amfoss/templeapp/activities/InsertData.java [hunk 1]
```diff
@@ -148,9 +148,14 @@ protected Void doInBackground(Void... params) {
             } catch (JSONException je) {
                 Log.i(Controller.TAG, getString(R.string.exception) + je.getLocalizedMessage());
             }
+            if (flag == 1) {
+                id = "REG";
+            }
```
Comment (yashk2000) [inline app/src/main/java/org/amfoss/templeapp/activities/InsertData.java]: shift this else to the line above. `} else {`

File: app/src/main/java/org/amfoss/templeapp/activities/InsertData.java [hunk 2]
```diff
@@ -184,9 +184,13 @@ protected Void doInBackground(Void... params) {
             } catch (JSONException je) {
                 Log.i(Controller.TAG, getString(R.string.exception) + je.getLocalizedMessage());
             }
+            if (flag == 1) {
+                id = "REG";
+            }else {
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/activities/InsertData.java]: `} else {` there is a space between `} else`

File: app/src/main/java/org/amfoss/templeapp/activities/InsertData.java [hunk 3]
```diff
@@ -148,9 +148,14 @@ protected Void doInBackground(Void... params) {
...
             }
+            if (flag == 1) {
+                id = "REG";
+            }
+            else {
+                id = "DON";
+            }
             return null;
         }
-
```
[the remark this answers is not in the document]
Comment (immadisairaj) [reply app/src/main/java/org/amfoss/templeapp/activities/InsertData.java]: ?

[the remark this answers is not in the document]
Comment (immadisairaj) [reply app/src/main/java/org/amfoss/templeapp/activities/InsertData.java]: > revert

what about this

**Comment by yashk2000** [comment]
@ShivangiSingh17 run `./gradlew spotlessApply` from the terminal please before sending a pr.

**Comment by immadisairaj** [comment]
> revert

what about this

**Pull request closed without merging**